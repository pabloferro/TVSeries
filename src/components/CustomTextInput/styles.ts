import {StyleSheet} from 'react-native';
import {white} from '../../constants/colors';

export default StyleSheet.create({
  textInput: {
    color: white,
    fontSize: 18,
  },
});
