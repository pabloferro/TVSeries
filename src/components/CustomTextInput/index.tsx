import React from 'react';
// eslint-disable-next-line no-restricted-imports
import {TextInput, TextInputProps} from 'react-native';

import {white} from '../../constants/colors';

import styles from './styles';

export default function CustomTextInput({style, ...props}: TextInputProps) {
  return (
    <TextInput
      placeholderTextColor={white}
      style={[styles.textInput, style]}
      {...props}
    />
  );
}
