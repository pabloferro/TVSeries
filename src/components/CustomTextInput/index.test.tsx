import React from 'react';
import {render} from '@testing-library/react-native';

import CustomTextInput from '.';

test('renders correctly', () => {
  const {toJSON} = render(<CustomTextInput />);

  expect(toJSON()).toMatchSnapshot();
});
