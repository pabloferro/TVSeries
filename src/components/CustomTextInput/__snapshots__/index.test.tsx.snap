// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TextInput
  placeholderTextColor="#FFF"
  style={
    Array [
      Object {
        "color": "#FFF",
        "fontSize": 18,
      },
      undefined,
    ]
  }
/>
`;
